lesson-10/public/app.js

```diff
@@ -0,0 +1,62 @@
+$('#button').click(() => {
+  const name = $('#myName').val();
+  const comment = $('#myComment').val();
+  $.ajax({
+    type: 'POST',
+    url: '/submit',
```

Comment:
Give url a more reasonable name. what kind of submit? Picture or text? What kind of picture? What kind of text?